hugr-core/src/export.rs

```diff
@@ -1029,6 +996,56 @@ impl<'a> Context<'a> {
         })
     }
 
+    fn export_value(&mut self, value: &'a Value) -> model::TermId {
+        match value {
+            Value::Extension { e } => {
+                let json = serde_json::to_string(e.value())
+                    .expect("extension values are always serializable");
+                let json = self.make_term(model::Term::Str(self.bump.alloc_str(&json)));
+                let runtime_type = self.export_type(&e.get_type());
+                let extensions = self.export_ext_set(&e.extension_reqs());
+                let args = self
+                    .bump
+                    .alloc_slice_copy(&[runtime_type, json, extensions]);
+                let symbol = self.resolve_symbol(TERM_JSON_CONST);
+                self.make_term(model::Term::ApplyFull { symbol, args })
+            }
+
+            Value::Function { hugr } => {
+                let outer_hugr = std::mem::replace(&mut self.hugr, hugr);
+                let outer_node_to_id = std::mem::take(&mut self.node_to_id);
+
+                let region = match hugr.root_type() {
+                    OpType::DFG(dfg) => {
+                        let extensions = self.export_ext_set(&dfg.extension_delta());
+                        self.export_dfg(hugr.root(), extensions, model::ScopeClosure::Closed)
+                    }
+                    _ => panic!("Value::Function root must be a DFG"),
+                };
+
+                self.node_to_id = outer_node_to_id;
+                self.hugr = outer_hugr;
+
+                self.make_term(model::Term::ConstFunc { region })
```

Comment:
no coverage? good to add something simple

Reply:
`Value::Function` can not yet be imported, so we can not roundtrip test this. I propose we delay that until we can import functions.